cmd/plugin/node_server.go

```diff
@@ -136,7 +140,7 @@ func (n NodeServer) NodePublishVolume(ctx context.Context, req *csi.NodePublishV
 		PullSecrets: req.Secrets,
 	}
 
-	if e := n.pullExecutor.StartPulling(po); e != nil {
+	if e := n.pullExecutor.StartPulling(po, valuesLogger); e != nil {
```

Comment:
Passing the values logger to startPulling and startMounting so they're able to log with values too.